css.css

```diff
@@ -0,0 +1,128 @@
+body {
+  background: rgb(188,86,188);
+background: radial-gradient(circle, rgba(188,86,188,1) 0%, rgba(150,182,255,1) 100%);
+}
+.container {
+  width: 1200px;
+  height: 100%;
+  margin: auto;
+  padding-top: 30px;
+}
+
+/* Search */
+.search {
+
+}
+
+.search h2 {
+  margin: 0;
+  margin-bottom: 10px;
+}
+
+
+
+/* Tests */
+#TestsAndResult {
+
+}
+
+.wrapper {
+  display: flex;
+}
+
+/* Arguments */
+#tests {
+  margin-right: 40px;
+}
+
+#testsList {
+  padding-left: 3px;
+  direction:rtl;
+  height: 500px;
+  width: 400px;
+  background-color: white;
+  overflow-x: hidden;
+  overflow-y: scroll;
+  /* white-space: nowrap; */
+}
+.testCase {
+  direction: ltr;
+}
+
+.testName {
+  height: 30px;
+  width: 400px;
+  display: flex;
+  background-color: white;
+  vertical-align: middle;
+  box-sizing: border-box;
+  border-bottom: 1px solid #44CFCB;
+}
+
+.test {
+  border-bottom: 1px solid purple;
+  width: 100%;
+  background-color: #55825d;
+  display: none;
+}
+
+.createTestCaseButton {
+ display: none;
+}
+input[type="text"] {
+    background: transparent;
+    border: none;
+}
+
+.createTestCaseButton img {
+  margin-left: 180px;
+  height: 30px;
+  width: 30px;
+}
+
+.testCaseArgument {
+  width: 50%;
+  min-height: 40px;
+  background-color: white;
+}
+.testCaseResult {
+  width: 50%;
+  min-height: 40px;
+  background-color: white;
+}
+.test .wrapper {
+  border-bottom: 5px solid white;
+}
+
+/* Results */
+#results {
+}
+
+#resultList {
+  height: 500px;
+  width: 400px;
+  background-color: white;
+}
+
+.result {
+  height: 80px;
+  width: 400px;
+  background-color: #fafafa;
+}
+
+/* addInputOutput */
+#createTestButton {
+  position: relative;
```

Comment:
Very bad example of using relative positioning here. Consider of flexbox or something.